tern.js

```diff
@@ -96,6 +203,48 @@ define(function(require, exports, module) {
             setDefEnabled: setDefEnabled,
             
             /**
+             * Sets tern server options
+             * @param {Object} ternServerOptions
+             */
+            setTernServerOptions: setTernServerOptions,
+            
+            /**
+             * Gets list of loaded tern definition names
+             * @param {tern~getTernDefNamesCallback} callback required function to retrieve names
+             */
+            getTernDefNames: getTernDefNames,
+            /**
+             * This callback is to retrieve names of definitions
+             * @callback tern~getTernDefNamesCallback
+             * @param {Array.String} names Array of names
+             */
+             
+             /**
+             * The complete Triforce, or one or more components of the Triforce.
+             * @typedef {Object} tern~pluginInfo
+             * @property {String} name - name of the plugin
+             * @property {boolean} enabled - Setting it false marks plugin for removal
+             * @property {String} path - Parameter to provide while loading a new plugin
+             */
```

Comment:
Ah, it's a typedef. Missed that. Please move it out of the members section then, above the freezePublicApi() call.